src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.Common.targets

```diff
@@ -45,32 +45,23 @@
       ReferencePaths="@(PackageValidationReferencePath)" />
   </Target>
 
-  <PropertyGroup>
-    <_GetReferencePathFromInnerProjectsDependsOn Condition="'$(IsCrossTargetingBuild)' != 'true'">_GetReferencePathForPackageValidation</_GetReferencePathFromInnerProjectsDependsOn>
```

Comment:
By making this an `MSBuild` call below will that result in an extra evaluation for a non-cross-targeting project?

Reply:
I provided the reasoning behind this change in the top post 🔝